pkg/acceptance/bettertestspoc/assert/objectassert/stream_snowflake_ext.go

```diff
@@ -38,3 +38,50 @@ func (s *StreamAssert) HasStageName(expected string) *StreamAssert {
 	})
 	return s
 }
+
+func (s *StreamAssert) HasSourceType(expected sdk.StreamSourceType) *StreamAssert {
+	s.AddAssertion(func(t *testing.T, o *sdk.Stream) error {
+		t.Helper()
+		if o.SourceType == nil {
+			return fmt.Errorf("expected source type to have value; got: nil")
+		}
+		if *o.SourceType != expected {
+			return fmt.Errorf("expected source type: %v; got: %v", expected, *o.SourceType)
+		}
+		return nil
+	})
+	return s
+}
+
+func (s *StreamAssert) HasBaseTables(expected []sdk.SchemaObjectIdentifier) *StreamAssert {
+	s.AddAssertion(func(t *testing.T, o *sdk.Stream) error {
+		t.Helper()
+		if o.BaseTables == nil {
+			return fmt.Errorf("expected base tables to have value; got: nil")
+		}
+		if len(o.BaseTables) != len(expected) {
+			return fmt.Errorf("expected base tables length: %v; got: %v", len(expected), len(o.BaseTables))
+		}
+		for i := range o.BaseTables {
+			if o.BaseTables[i].FullyQualifiedName() != expected[i].FullyQualifiedName() {
```

Comment:
Shouldn't it be contains instead of relying on order?

Reply:
Done.